brambling/templates/brambling/event/order/attendee_basic_data.html

```diff
@@ -33,57 +33,123 @@ <h3 class="margin-leader-0">
 						</div>
 					</div>
 
 					<div class="row collapse" id="extraName">
 						<div class="col-sm-6">
-							{% formrow form.middle_name %}
+							{% formrow basic_data_form.middle_name %}
 						</div>
 						<div class="col-sm-6">
-							{% formrow form.name_order %}
+							{% formrow basic_data_form.name_order %}
 						</div>
 					</div>
 
 					<a href="#extraName" data-toggle="collapse" class="{% if masthead %}masthead-link {% endif %}pull-right more-name-options"><small><i class="fa fa-plus"></i> More Name Options</small></a>
-					{% formrow form.email %}
-					{% formrow form.phone %}
+					{% formrow basic_data_form.email %}
+					{% formrow basic_data_form.phone %}
 
-					{% formrow form.items %}
+					{% formrow basic_data_form.items %}
 
-					{% if form.housing_status %}
-						{% formrow form.housing_status %}
+					{% if basic_data_form.housing_status %}
+						{% formrow basic_data_form.housing_status %}
 					{% endif %}
 
-					{% for field in form.custom_fields %}
+					{% for field in basic_data_form.custom_fields %}
 						{% formrow field %}
 					{% endfor %}
 
-					{% formrow form.photo_consent %}
+					{% formrow basic_data_form.photo_consent %}
 
 					<div class="well">
 						{{ event.get_liability_waiver }}
-						{% formrow form.liability_waiver with label="I have read and agree to the waiver above." %}
+						{% formrow basic_data_form.liability_waiver with label="I have read and agree to the waiver above." %}
 					</div>
 
 				{% endform %}
+
+				{% if event.collect_housing_data %}
+					<div id='housing_form' class='panel panel-default'>
+						<header class='panel-heading'>
+							<div class='panel-title'>
+								Housing
+							</div>
+						</header>
+						<div class='panel-body'>
+							{% form housing_form using %}
+								<div class='row'>
+									<div class='col-md-6'>
+										{% formrow housing_form.nights %}
+									</div>
+									<div class='col-md-6'>
+										{% formrow housing_form.housing_prefer %}
+									</div>
+								</div>
+
+								<div class='row'>
+									<div class='col-md-6'>
+										{% formrow housing_form.ef_cause %}
+										{% if housing_form.ef_cause_confirm %}{% formrow housing_form.ef_cause_confirm %}{% endif %}
+									</div>
+									<div class='col-md-6'>
+										{% formrow housing_form.ef_avoid %}
+										{% if housing_form.ef_avoid_confirm %}{% formrow housing_form.ef_avoid_confirm %}{% endif %}
+									</div>
+								</div>
+
+								<div class='row'>
+									<div class='col-md-6'>{% formrow housing_form.person_prefer %}</div>
+									<div class='col-md-6'>{% formrow housing_form.person_avoid %}</div>
+								</div>
+
+								{% formrow housing_form.other_needs %}
+
+								{% for field in housing_form.custom_fields %}
+									{% formrow field %}
+								{% endfor %}
+
+								{% formrow housing_form.save_as_defaults %}
+
+							{% endform %}
+						</div>
+					</div>
+				{% endif %}
```

Comment:
Looks good! :+1: I'd prefer to avoid panels, but I agree that with the current design, it's important to make sure these questions are clearly grouped.